### PR TITLE
test: provided mock responses from server for SASVIYA + SASJS

### PR DESCRIPTION
**Pull request by saadjutt01**
## Issue

no issue to link.

## Intent

Remove dependency of `SASJS` server for specs.
Update specs of testing using `SASJS` server.

## Implementation

Updated, test for both `SASJS` and `SASVIYA`.
Provided mocked responses
Removed deploy steps.
Converted `testing.server.spec.ts` to `testing.spec.ts`

## Checks

- [ ] Code is formatted correctly (`npm run lint:fix`).
- [ ] Any new functionality has been unit tested.
- [ ] All unit tests are passing (`npm test`).
- [ ] All CI checks are green.
- [ ] JSDoc comments have been added or updated.
- [ ] Reviewer is assigned.

### CODE REVIEW

File: src/commands/testing/test.ts [hunk 1]
```diff
@@ -258,15 +180,89 @@ export async function runTest(
...
+          displayError(
+            {},
+            `'test_results' not found in server response, to debug click ${testUrl}`
+          )
+
+          printCodeExample()
         }
+      })
+      .catch(async (err) => {
         printTestUrl()
```
Comment (saadjutt01) [inline src/commands/testing/test.ts:263]: This major change is nothing, just converted named callback function `handleRes` to unnamed for `.then`.

Comment (sabhas) [reply src/commands/testing/test.ts:263]: Can you provide a line of explanation about why you changed `handleRes` callback function to unnamed in then block?

File: src/utils/createSASjsInstance.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import SASjs, { SASjsConfig } from '@sasjs/adapter/node'
+
+export const createSASjsInstance = (config: Partial<SASjsConfig>): SASjs => {
+  console.log('actual called')
```
Comment (sabhas) [inline src/utils/createSASjsInstance.ts:4]: console log statement should be removed.

File: src/commands/testing/test.ts (hunk 1 above)
Comment (saadjutt01) [reply src/commands/testing/test.ts:263]: It was redundant, to have named callback function for then.

It was added by me few days ago( in order to have similar behaviour for SASJS) , in order to handle a case, but we configured adapter to return correct response.

File: src/utils/createSASjsInstance.ts (hunk 2 above)
Comment (saadjutt01) [reply src/utils/createSASjsInstance.ts:4]: Good catch.

**Review by sabhas — APPROVED** [review]

**Pull request merged**